pkgs/tools/misc/pista/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, rustPlatform, fetchFromGitHub, openssl, pkg-config }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pista";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "NerdyPepper";
+    repo = "pista";
+    rev = "v${version}";
+    sha256 = "1nrqgl5r5ja95g5xhfvkvfz72g318pqc7yj4fwl6xnad19smvms0";
```

Comment:
```suggestion
    sha256 = "0fbvi85vlh7pk0f6d2q05na2jpbgwpylwzzms6syvywk8dm9hd1f";
```

Reply:
Seems to break the build. Setting `fetchSubmodules` to `true` works though. Not sure how the build succeeded previously.